lib/Optimizer/Transforms/Utils.cpp

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+#include "cudaq/Optimizer/Dialect/CC/CCOps.h"
+
```

Comment:
nit: blank line between include directives.

Reply:
Removed this file.